ui/tests/acceptance/auth/auth-list-test.js

```diff
@@ -83,40 +84,48 @@ module('Acceptance | auth backend list', function (hooks) {
       .map((backend) => backend.type)
       .forEach((type) => {
         test(`${type} auth method`, async function (assert) {
-          const supportManaged = supportedManagedAuthBackends();
-          const path = type === 'token' ? 'token' : `auth-list-${type}-${this.uid}`;
-          if (type !== 'token') {
-            await enablePage.enable(type, path);
```

Comment:
this was the method that I swapped out. if we have semi-regular flaky behavior and page objects are involved, I head there first. I removed the nested use of them by replacing this function that relied on page objects.